### DISCUSSION TITLE
Redirect local storage of clips in Hassio (subdir in media)

### DISCUSSION THREAD

**Discussion by Moondevil-ha** [blakeblackshear/frigate#1940] [General]
*ignore* still troubleshooting the mount points

**Comment by blakeblackshear** [comment]
Many have been down this path, and I haven't seen anyone get it to work without a supervised install instead of HassOS.

**Reply by Moondevil-ha** [reply]
Managed to get it working via some udev code I found in a different thread. Mounting the SATA drive to Media in the HA terminal via a rules.d entry *appears* to work, and be persistant. Firgate is now writing to the SATA drive, evidenced by the increasing storage usage on it, and the SSD NOT increasing. Obviously not something you can work in to Frigate, but I might stick a request in to HassOS for an RFE. Thread is here: https://community.home-assistant.io/t/solved-mount-usb-drive-in-hassio-to-be-used-on-the-media-folder-with-udev-customization/258406/46?u=moondevil-ha
